filebeat/f.yml

```diff
@@ -0,0 +1,143 @@
+###################### Filebeat Configuration Example #########################
```

Comment:
that was probably commited by accident. You can use filebeat.dev.yml and it will be ignored automatically by gitignore.